easyrsa3/vars.example

```diff
@@ -220,3 +220,8 @@ fi
 
 #set_var EASYRSA_BATCH		""
 
+# These two variables allow the user to specify a source for password,
+# using any openssl password options like pass:1234 or env:var
+
+#set_var EASYRSA_PASSIN		""
```

Comment:
I don't see why someone would set this by default. passin/passout have different meanings for different commands. I would remove those lines.

Reply:
I meant this only for a reference / documentation. It's true that these variables are empty / not set by default, what is shown in that file.

The command `build-ca` requires both env vars to be set, and it makes sense to set it to the same `env:var` value.

Anyway, I just wanted to document it as best I could, but if you still think that this is not appropriate I can remove it. Please confirm. Thanks.

Reply:
I've dropped the examples from config file.